### DISCUSSION TITLE
DBEMT

### DISCUSSION THREAD

**Discussion by Unique-Wang** [OpenFAST/openfast#1167] [Q&A] [answered]
Dear @jjonkman ,

I really  want to find the document to help me better understand BEMT and DBEMT ,I found this on the forum https://forums.nrel.gov/t/dynamic-blade-element-momentum-theory-dbemt/2174, is the full paper published yet? If already published,where can I find it?

Best regards,

**Comment by ebranlard** [comment, marked as answer]
Dear @Unique-Wang 

The [final paper](https://iopscience.iop.org/article/10.1088/1742-6596/2265/3/032044) is now published, it presents a continuous-time state-space formulation of DBEMT, which will be available in the next release of OpenFAST (as soon as [this pull request](https://github.com/OpenFAST/openfast/pull/1000) is merged). The model is equivalent to the discrete-time implementation which is currently the only option available in OpenFAST. Note that this state-space model is slightly different than the one presented in Section B.3 of the [document mentioned in the post](https://forums.nrel.gov/uploads/short-url/Ao4OuvD6ay00IuDBdbwjYFvolnw.pdf). This document is yet useful to understand the discrete-time formulations.  

DBEMT will be documented in the readthedocs documentation of OpenFAST in the next release, under User Documentation > Modules > AeroDyn > AeroDyn Theory > DBEMT. 

To understand BEM theory, we do not have an updated theory manual for AeroDyn 15, but you can refer to the previous [AeroDyn Manual](https://www.nrel.gov/docs/fy05osti/36881.pdf).

Best regards,
Emmanuel

**Reply by Unique-Wang** [reply]
Thanks！
